ibm_catalog.json

```diff
@@ -0,0 +1,120 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-slz-was-operator",
```

Comment:
I dont think we can use `slz` in the name, right @SirSpidey ?

Reply:
scrap that, its used in our SLZ Das

Reply:
we definitely don't want to show `SLZ` in descriptions. Refer to it either as `the landing zone deployable architectures` or the real name of the OCP DA (Red Hat OpenShift Container Platform on VPC landing zone).